assets/components/molecules/gallery/gallery.scss

```diff
@@ -0,0 +1,166 @@
+@charset 'utf-8';
+
+$gallery-btn-size: 41px;
+$gallery-btn-margin: 14px;
+
+.gallery,
+.gallery-nav {
+  position: relative;
+  opacity: 0;
+  transition: opacity 0.3s;
+  // [class*="-item"]:not(:first-child) { display: none; }
+
+  &.ready {
+    opacity: 1;
+    // [class*="-item"]:not(:first-child) { display: block; }
+  }
+
+  .img-fluid {
+    width: 100%;
+  }
+}
+
+.gallery-nav:after {
+  content: ' ';
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  width: 5%;
+  background: linear-gradient(to right, transparent 0%, $white 100%);
+}
+
+.gallery.flickity-enabled.is-fullscreen {
+  background: $white;
+
+  .flickity-slider {
+    display: flex;
+    align-items: center;
+  }
+
+  .gallery-item {
+    align-items: center;
+    max-height: 80vmin;
+
+    .img-fluid {
+      max-width: 100vmin;
+      margin: 0 auto;
+    }
+
+    figcaption span:last-child { padding-left: 3rem; }
+  }
+
+  .flickity-prev-next-button {
+    top: 50% !important;
+    border: 1px solid $gray-300;
+    &.next { right: 6%; }
+    &.previous { left: 6%; }
+  }
+}
+
+.gallery-item {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  figcaption {
+    display: flex;
+    justify-content: space-between;
+    padding: 0.6rem 0.2rem;
+    font-size: $font-size-sm;
+    color: $gray-600;
+
+    > span {
+      display: inline-block;
+      flex: 1 1 auto;
+    }
+
+    > span:last-child {
+      text-align: right;
+      padding-left: 1rem;
+    }
+  }
+}
+
+.gallery-nav-item {
+  display: block;
+  position: relative;
+  width: calc(23% - 0.8rem);
+  margin-right: 0.8rem;
+  border-bottom: 1px solid $gray-300;
+
+  &:after {
+    content: ' ';
+    position: absolute;
+    top: 0;
+    left: 50%;
+    margin-left: -15px;
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 15px 15px 0;
```

Comment:
Could `$grid-gutter-width / 2` be used instead of `15px` or has this nothing to do with the general spacing?